projects/plugins/jetpack/changelog/add-cpt-menu-calypso-menu

```diff
@@ -0,0 +1,5 @@
+Significance: minor
+Type: enhancement
+Comment: Include custom post types items inside the calypso admin menu
```

Comment:
The changelog cannot be omitted now that this is a minor change.
```suggestion

Include custom post types items inside the calypso admin menu
```

Reply:
Oh!, thank you!, I thought the Significance type was what made the "comment" public or not.